tests/content_types_test.py

```diff
@@ -17,6 +17,21 @@ def find_one(self, req, **lookup):
         }
 
 
+class MockVocabulariesService():
+
+    def get_custom_vocabularies(self):
+        return [
+            {
+                "_id": "test",
+                "field_type": None,
+                "selection_type": "do not show",
```

Comment:
seems like this is bool value, would be probably more efficient to store it as such.
also the name is a bit misleading imo, it seems like there is some selection type like radio/checkboxes/select for the field, while this controls if field is displayed

Reply:
There are 3 values for selection_type: 'single selection', 'multi selection' and 'do not show'

Reply:
so what does it mean actually?

Reply:
https://dev.sourcefabric.org/browse/SDESK-3551

if I select a "single value" type I expect the CV to be shown as a dropdown in the UI
if I select a "multiple value", I expect the CV to be shown as a multi-select UI component
Should disable the option to add to CPs the CVs that had set 'do not show' value

Reply:
ok I see, if there should be single control in the cv editor for all these it makes sense. can you pls add the field to schema and list those as allowed values with some comment what it does?

Reply:
It is already added to vocabularies schema:
https://github.com/superdesk/superdesk-core/blob/master/superdesk/vocabularies/vocabularies.py#L81